src/components/Hero.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faBars } from '@fortawesome/free-solid-svg-icons';
+import { StyledHero } from 'styles/Hero.styles';
+import { CallButton } from './CallButton';
+
+export const Hero = () => {
+  return (
+    <StyledHero>
+      <div className="nav-bar">
```

Comment:
Is the hamburger menu supposed to be there in desktop mode also or should it be replaced by a bigger navbar-menu?